packages/frontend-2/components/settings/shared/projects/index.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+  <div>
+    <div class="flex flex-col-reverse md:justify-between md:flex-row md:gap-x-4">
+      <div class="relative w-full md:max-w-md mt-6 md:mt-0">
+        <FormTextInput
+          name="search"
+          :custom-icon="MagnifyingGlassIcon"
+          color="foundation"
+          search
+          :show-clear="!!search"
```

Comment:
showClear doesn't have to be conditional, it already disappears if there's no content

Reply:
Thanks, adjusted this :)